src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesListProps = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<MoviesListProps> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => <MovieCard movie={movie} key={movie.imdbId} />)}
+  </div>
```

Comment:
```suggestion
  <div className="movies">
    {movies.map((movie) => (
      <MovieCard movie={movie} key={movie.imdbId} />
    ))}
  </div>
```

and use semantic tags for list rendering like `ul` and `li`